lib/zen_ex/core/models/ticket.ex

```diff
@@ -14,10 +14,10 @@ defmodule ZenEx.Model.Ticket do
   ## Examples
 
       iex> ZenEx.Model.Ticket.list
-      %ZenEx.Collection{}
+      {:ok, %ZenEx.Collection{}}
 
   """
-  @spec list :: %ZenEx.Collection{} | {:error, String.t()}
+  @spec list :: {:ok, %ZenEx.Collection{}} | {:error, any()}
```

Comment:
It would be nice to create [User-defined types](https://hexdocs.pm/elixir/1.12/typespecs.html#user-defined-types) and prevent [Credo.Check.Warning.SpecWithStruct](https://hexdocs.pm/credo/Credo.Check.Warning.SpecWithStruct.html)